packages/client/src/execution/vmexecution.ts

```diff
@@ -160,21 +165,23 @@ export class VMExecution extends Execution {
     const stateManager = new DefaultStateManager({
       trie,
       prefixStorageTrieKeys: this.config.prefixStorageTrieKeys,
-      accountCacheOpts: {
-        deactivate: false,
-        type: CacheType.LRU,
-        size: this.config.accountCache,
-      },
-      storageCacheOpts: {
-        deactivate: false,
-        type: CacheType.LRU,
-        size: this.config.storageCache,
-      },
-      codeCacheOpts: {
-        deactivate: false,
-        type: CacheType.LRU,
-        size: this.config.codeCache,
-      },
+      caches: new Caches({
+        accountCacheOpts: {
```

Comment:
I think we can fully drop the `CacheOpts` from this key, makes it nicer to digest/read and remains fully expressive.

Reply:
(same for others of course)

Reply:
done!